services/camera.service.ts

```diff
@@ -73,19 +80,22 @@ export default class CameraService implements Services.ServiceInstance {
 
         if (cameraSource) {
           const defaultCameraFeedPath = path.resolve(cameraSource);
-          if (!fs.existsSync(defaultCameraFeedPath)) {
-            throw new Error(`Default camera feed ${defaultCameraFeedPath} does not exist`);
-          }
-
           const sourceCameraFeedPath = path.resolve(process.cwd(), videoPath);
           if (!fs.existsSync(sourceCameraFeedPath)) {
             throw new Error(`New source camera feed ${sourceCameraFeedPath} does not exist`);
           }
+          if (this.browser?.requestedCapabilities.browserName?.toLowerCase().includes('chrome') && !this.browser?.requestedCapabilities.platformName?.toLowerCase().includes('android')) {
```

Comment:
Let's use `capabilities` instead of `requestedCapabilities` since these are more reliable values by the driver.
```suggestion
          if (this.browser?.capabilities.browserName?.toLowerCase().includes('chrome') && !this.browser?.capabilities.platformName?.toLowerCase().includes('android')) {
```

Reply:
Thanks, Applied changes